core/go/pkg/persistence/testdb_sqlite.go

```diff
@@ -46,19 +42,3 @@ func NewUnitTestPersistence(ctx context.Context, suite string) (Persistence, fun
 	})
 	return p, func() { p.Close() }, err
 }
-
-func TestMigrateUpDown(t *testing.T) {
```

Comment:
These tests weren't actually being run - and I found some issues with broken `down` migrations once I re-instated them.